xblock/plugin.py

```diff
@@ -139,7 +138,7 @@ def load_classes(cls, fail_silently=True):
         """
         all_classes = itertools.chain(
             pkg_resources.iter_entry_points(cls.entry_point),
-            (entry_point for identifier, entry_point in cls.extra_entry_points),
+            (entry_point for identifier, entry_point in iter(cls.extra_entry_points)),
```

Comment:
why you are doing this ?

Reply:
This is a lazy member of our class which returns a list when called, but pylint doesn't consider it an iterable and warns, so wrapping it in `iter()` does the trick.